deployments/prod_addresses.json

```diff
@@ -3,7 +3,6 @@
     "SignatureVerifier": "0x80e51327A393e6d6777c85c9AC16baD45086ca5C",
     "Hasher": "0xDf4fF02E2dDe3A08590829d7398Cc31B0255bAb5",
     "CapacitorFactory": "0x66AD32F5F3C42C6B8102e998f24936F57997cE18",
-    "GasPriceOracle": "0x09b471B83b91bdb1AA361f05770CfcEFF7526b65",
```

Comment:
Lets not delete these until we do a prod deployment and actually stop using it it.